docs/guide/pmndrs/glitch.md

```diff
@@ -4,6 +4,10 @@
   <GlitchDemo />
 </DocsDemo>
 
+::: info
+This effect extends the [GlitchEffect](https://pmndrs.github.io/postprocessing/public/docs/class/src/effects) from the [postprocessing](https://pmndrs.github.io/postprocessing/public/docs/index.html) library.
+:::
+
```

Comment:
**question:** What do you mean with that? We are not really extending it. It is just wrapped.

Reply:
You are right, I will change the wording

Reply:
Actually I will remove it, none of the other guides have something similar and its a little bit redundant